pkg/reconciler/mtbroker/trigger/trigger_test.go

```diff
@@ -352,6 +353,12 @@ func TestReconcile(t *testing.T) {
 					WithTriggerDependencyReady()),
 			}},
 			WantDeletes: []clientgotesting.DeleteActionImpl{{
+				ActionImpl: clientgotesting.ActionImpl{
+					Namespace: testNS,
+					Resource: schema.GroupVersionResource{
```

Comment:
This should probably be something like `v1.SchemeGroupVersion.WithResource("subscriptions")`

Reply:
serving had those as strings, so i just did that

Reply:
What I gave produces the full GVR, you only provide what the test fwk looks at today. 😇